core/src/main/scala/org/apache/spark/sql/crossdata/security/DefaultSecurityManager.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.crossdata.security
+
+class DefaultSecurityManager(override val credentials: Credentials) extends SecurityManager(credentials) {
+
+  val user = credentials.user match {
```

Comment:
That pattern matching is equivalent to a map which receives the identity function as parameter. That is, as @darroyocazorla hints, it doesn't perform any transformation.